tests/blackbox_test.go

```diff
@@ -359,6 +321,51 @@ func outer(t *testing.T, test types.Test, negativeTests bool) error {
 	return nil
 }
 
+// Replace all UUID variables (format $uuid<num>) in configs and partitions with an UUID
+func replaceAllUUIDVars(t *testing.T, test types.Test, UUIDmap map[string]string) (types.Test, error) {
```

Comment:
UUIDmap can be generated in this function now, right? Nothing outside this function depends on it? This also doesn't need to take `t` (since replaceUUIDVars doesn't either).

Reply:
I'd even go as far as to make this a method for the test struct (make sure to use a pointer receiver) and just call `test.ReplaceUUIDs` or something

Reply:
Added a pointer receiver. I'm not sure if there is a more efficient way than using a struct since type.Test is in a different package. If there is, it might be beneficial to also do this for replaceAllUUIDVarsInParitions() but have a types.Disk pointer receiver.

Reply:
Given that the test also needs to replace the uuids in the config as well, I'd prefer to keep it defined on `types.Test`. This should be moved to the types package to avoid the try redefinition.